app/_src/using-mesh/managing-ingress-traffic/builtin.md

```diff
@@ -78,6 +78,200 @@ kuma-dp run \
   --dataplane-file=my-gateway.yaml # the Dataplane resource described above
 ```
 
+{% endtab %}
+{% tab setup Kubernetes without MeshGatewayInstance %}
+
+Using `MeshGatewayInstance` is highly recommended. If for any reason you are
+unable to use `MeshGatewayInstance` to deploy builtin gateways, you can manually create a `Deployment` and `Service`
+to manage `kuma-dp` instances and forward traffic to them.
+Keep in mind however, that you'll need to keep the listeners of your
+`MeshGateway` in sync with your `Service`.
+
+These instructions will use the resources created by `MeshGatewayInstance` as a
+guide but it's likely worth creating a `MeshGatewayInstance` to configure as
+much as you can and using it as a basis for your resources.
+
+Given a `MeshGateway` spec:
+
+```yaml
+spec:
+  conf:
+    listeners:
+    - port: 80
+      protocol: HTTP
+  selectors:
+  - match:
+      kuma.io/service: demo-app_gateway
+```
+
+#### `Service`
+
+The `Service` will forward traffic to the `kuma-dp` we'll configure in the next
+section. Its `ports` need to be in sync with the `MeshGateway` `listeners`.
+
+```yaml
+metadata:
+  annotations:
+    kuma.io/gateway: builtin
+  name: demo-app-gateway
+  namespace: kuma-demo
+spec:
+  ports:
+  - name: "80"
+    port: 80
+    protocol: TCP
+    targetPort: 80
+  selector:
+    app: demo-app-gateway
+```
+
+The `selector` should match the `Pod` template created in the next section.
+
+#### `Deployment`
+
+The `Deployment` we'll create will manage running some number of `kuma-dp`
+instances that are configured to serve traffic for your `MeshGateway`.
+
+We'll cover just `spec.selector` and `spec.template` here because
+other parts of the `Deployment` can be configured arbitrarily.
+Most importantly you'll need to change:
+
+* `kuma.io/tags` annotation - must match the `MeshGateway` `selectors`
+* `KUMA_CONTROL_PLANE_CA_CERT` environment variable - can be retrieved with `kubectl get secret kuma-tls-cert  -n kuma-system -o=jsonpath='{.data.ca\.crt}' | base64 -d`
```

Comment:
I used `mesh_product_name_path` because I don't think we have something specific to "part of resource name" (?)

```suggestion
* `KUMA_CONTROL_PLANE_CA_CERT` environment variable - can be retrieved with `kubectl get secret {{site.mesh_product_name_path}}-tls-cert -n {{site.mesh_namespace}} -o=jsonpath='{.data.ca\.crt}' | base64 -d`
```